<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spezialmodule</class>
 <widget class="QDialog" name="Spezialmodule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Spezialmodule</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_Spezial">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>910</width>
       <height>230</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>830</width>
       <height>236</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Spezialmodule</string>
     </property>
     <widget class="QLabel" name="lb_mod">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Spezialmodul</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_beschr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Beschreibung/Anmerkung</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Modul">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>531</width>
        <height>22</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_Beschreib">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>531</width>
        <height>81</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Suchen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>102</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Suchen</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_loschen">
      <property name="geometry">
       <rect>
        <x>346</x>
        <y>200</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Löschen</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_report">
      <property name="geometry">
       <rect>
        <x>692</x>
        <y>200</y>
        <width>102</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Report</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_excel">
      <property name="geometry">
       <rect>
        <x>584</x>
        <y>200</y>
        <width>102</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Excel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_hilfe">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>200</y>
        <width>102</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hilfe</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/info_13213.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/info_13213.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_abbrechen">
      <property name="geometry">
       <rect>
        <x>453</x>
        <y>200</y>
        <width>125</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hauptmenü</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_speichern">
      <property name="geometry">
       <rect>
        <x>226</x>
        <y>200</y>
        <width>115</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Speichern</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/save_3621.ico</normaloff>C:/Users/glaser3/_Symbole/UsedSymbols/save_3621.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>200</y>
        <width>102</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>100</horstretch>
        <verstretch>25</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Leeren</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../_Symbole/UsedSymbols/-clear_90704.ico</normaloff>../../_Symbole/UsedSymbols/-clear_90704.ico</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tw_Table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>910</width>
       <height>460</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lab_INFO">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
